torch/utils/__init__.py

```diff
@@ -46,6 +46,27 @@ def swap_attr(name):
         setattr(t1, name, (getattr(t2, name)))
         setattr(t2, name, tmp)
 
+    def error_pre_hook(grad_outputs):
+        raise RuntimeError("Trying to execute AccumulateGrad node that was poisoned by swap_tensors")
```

Comment:
Would be nice to mention some way for the user to rectify the issue assuming that the user ran into this issue via _apply?